### DISCUSSION TITLE
Full screen home on tablet

### DISCUSSION THREAD

**Discussion by farridav** [DigiLive/mushroom-strategy#150] [Q&A]
Hi, 

First off, thanks for the amazing project.

I have this running on my wall tablet, and the home view is still rendering like it would on a phone.

Is there any way to make the home view fluid or fill the screen somehow ? Looks great on a phone but not so much on a tablet (until you open an area)

**Comment by DigiLive** [comment]
I think this is possible since release 2024.9 of Home Assistant by using sections.
https://www.home-assistant.io/blog/2024/09/04/release-20249/#sections-go-big

However, this will still need coding from our side.
It would help if you manually setup a [Mushroom](https://github.com/piitaya/lovelace-mushroom) dashboard and share that yaml configuration with us.
